### DISCUSSION TITLE
Question about extending editor with new custom widgets

### DISCUSSION THREAD

**Discussion by michal-tobiasz-wttech** [wcm-io/io.wcm.caconfig.editor#15] [Q&A] [answered]
Does the frontend part of the editor provides extension points for providing new custom widgets like for example date picker?

**Comment by stefanseifert** [comment, marked as answer]
no. the frontend JS is build in a modular way, but currently not designed to be extended by custom widgets.
however, widgets like a date picker sound quite generic, so if you come up with a PR for it, we can included it in the main code base.

(unfortunately the editor frontend is implemented in Angular JS 1.x, which is quite outdated nowadays.)
